### PR TITLE
Fix the link to the table with mem_advice and prefetch method descriptions.

### PR DESCRIPTION
**Pull request by bader**

### CODE REVIEW

File: adoc/chapters/programming_interface.adoc [hunk 1]
```diff
@@ -10187,7 +10187,7 @@ If a device supports concurrent access to shared allocations, then
 Additionally, users may use the [code]#mem_advise# member function to annotate
 shared allocations with [code]#advice#.
 Valid [code]#advice# is defined by the device and its associated backend.
-See <<table.members.queue>> and <<table.members.handler.copy>> for more
+See <<table.queue.shortcuts>> and <<table.members.handler.copy>> for more
```
Comment (gmlueck) [inline adoc/chapters/programming_interface.adoc:10190]: Thanks for fixing this.  However, the same error exists in the paragraph above too.  Can you fix that one also?

Comment (bader) [reply adoc/chapters/programming_interface.adoc:10190]: Updated the link for prefetch method as well.

**Review by gmlueck — APPROVED** [review]
LGTM.

**Comment by gmlueck** [comment]
Merging as editorial.

**Pull request merged**